### DISCUSSION TITLE
Can it be redirected to the page that the user tried to enter once it has been logged in?

### DISCUSSION THREAD

**Discussion by JuandaGarcia** [gladly-team/next-firebase-auth#197] [Q&A] [answered]
Hello, I would like to know if this functionality is possible with the library.

An example can be how stripe does it.

When you try to enter this link and you do not have an active session, it will redirect you to the login

[https://dashboard.stripe.com/test/payments](https://dashboard.stripe.com/test/payments)

![Captura de Pantalla 2021-06-07 a la(s) 5 52 35 p  m](https://user-images.githubusercontent.com/39936759/121097289-5e4e5080-c7b9-11eb-932c-3d5cbf2db3c4.png)

And once you log in, it will redirect you to the page you tried to access previously

![Captura de Pantalla 2021-06-07 a la(s) 6 02 05 p  m](https://user-images.githubusercontent.com/39936759/121098050-ad48b580-c7ba-11eb-8dce-64bfeed8bbf0.png)

**Comment by kmjennison** [comment, marked as answer]
Yes. See an example here: https://github.com/gladly-team/next-firebase-auth/issues/122#issuecomment-815390638

**Reply by JuandaGarcia** [reply]
Thank you so much
